modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/package.json

```
@@ -36,6 +36,7 @@
 		"graphql": "^16.0.1",
 		"react": "^17.0.2",
 		"react-dom": "^17.0.2",
+		"react-router-dom": "6",
```

@iagochaves can you upgrade to the latest version?

Probably, not is going to break

```suggestion
		"react-router-dom": "^6.2.1",
```